packages/reactive_ble_mobile/ios/Classes/ReactiveBle/Central.swift

```diff
@@ -275,6 +277,22 @@ final class Central {
         
         return response
     }
+    
+    func writeDescriptorWithoutResponse(
+        value: Data,
+        descriptor qualifiedDescriptor: QualifiedDescriptor
+    ) throws {
+        let descriptor = try resolve(descriptor: qualifiedDescriptor)
+        
+        guard let characteristic = descriptor.characteristic else {
+            throw Failure.descriptorNotFound(qualifiedDescriptor)
+        }
+
+        guard let response = characteristic.service?.peripheral?.writeValue(value, for: descriptor)
```

Comment:
@ened Is writing to a descriptor inherently a "without response" (without acknowledgement) operation? If so, then using `-WithoutResponse` is reasonable, but please clarify.

Reply:
Writing to descriptors works via https://developer.apple.com/documentation/corebluetooth/cbperipheral/1519107-writevalue . There is no response per se, but the docs at https://developer.apple.com/documentation/corebluetooth/cbperipheraldelegate/1519062-peripheral state that a delegate method will be called once the write is completed. If it fails, the error is given as well. Do you think the plugin should implement that delegate and return a value only once that is done?

`WithoutResponse` still applies based on that logic, but I am wondering what the plugin should do.

Reply:
> Do you think the plugin should implement that delegate and return a value only once that is done?

If waiting for the events delivered to the delegate adds anything, it might be a potential improvement, but at of now, I'd say, this method should be implemented similar to https://github.com/PhilipsHue/flutter_reactive_ble/blob/d3d278163e67978c3df9ab2fe7e890f840cb23fd/packages/reactive_ble_mobile/ios/Classes/ReactiveBle/Central.swift#L264

As long as they match, it should be enough.